src/convertToObject.js

```diff
@@ -2,17 +2,24 @@
 
 /**
  * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').filter(clear => clear.trim())
+    .reduce((acc, string) => {
+      const [command, value] = string.split(':');
+
+      acc[command.trim()] = value.trim();
+
+      return acc;
+    }, {});
```

Comment:
[checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 4th item